.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+  tests: 
+    name: Tests
+    runs-on: ubuntu-20.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Let's keep it ubuntu-latest

Reply:
@2002Bishwajeet 
Travis runs ubuntu-20.04. (https://github.com/utopia-php/cache/blob/master/.travis.yml#L1)
Should I change to `ubuntu-latest`???

Reply:
Yep

Reply:
@2002Bishwajeet Done